tests/admin_views/tests.py

```diff
@@ -3658,6 +3665,10 @@ def test_simple_inline(self):
         response = self.client.get(collector_url)
         self.assertContains(response, 'name="widget_set-0-id"')
 
+        # No file or image fields, no enctype on the forms
+        self.assertIs(response.context['has_file_field'], False)
+        self.assertNotContains(response, 'enctype="multipart/form-data"')
```

Comment:
I'd make `'enctype="multipart/form-data"'` a constant or class attribute to avoid typos since `assertNotContains` is fragile with respect to that.

Reply:
Right, so that if the `assertContains()` ever changes, the "not contains" version will also be updated.